common/source/docs/common-here-plus-gps.rst

```diff
@@ -11,11 +11,6 @@ These instructions focus on the setup using Mission Planner on Windows, but HERE
 .. image:: ../../../images/here-plus-gps.png
 	:target: ../_images/here-plus-gps.png
 
-Where to buy
```

Comment:
Might be better to update this to say something along the lines of: Newer generation of this product can be purchased through CubePilot resellers at https://cubepilot.org/#/reseller/list